streamwebs_frontend/streamwebs/templates/streamwebs/faq.html

```diff
@@ -0,0 +1,69 @@
+{% extends 'streamwebs/base.html' %}
+{% load staticfiles %}
+{% load i18n %}
+
+{% block content %}
+<style>
+  ul.list li{
+    list-style-type: square; !important
+  }
+</style>
+<h4 class="teal-text center">{% trans "Frequently Asked Questions" %}</h4>
+<br/>
+</br/>
+
+<div class="container">
+<h5 class="teal-text">How do I access my old account?</h5>
+<p>
+    You will receieve an email to the account you signed up with on the original
+    site. From there you can reset your password and use the same account on
+    the new site!
```

Comment:
You should add something like: "If you didn't receive an email, you can also reset your password by clicking on "Login" and then "Forgot your password?" and input the email you registered to the site.